docs/platform/api_keys_and_pat_tokens.mdx

```diff
@@ -0,0 +1,84 @@
+---
+id: api-keys-and-pat-tokens
+title: API keys and PAT tokens
+sidebar_label: API keys and PAT tokens
+slug: /platform/api-keys-and-pat-tokens
+---
+
+API keys and PAT tokens are authentication methods enabling you to manage your data via REST API.
+
+## Project API keys
+:::info
+The API key format was changed in version **2.38.0**. Now all API keys have the `tgpak_` prefix. However, if you provide
+legacy API key to the header or to query param, it will be accepted.
+:::
+
+
+Project API keys are useful to manipulate project resources like keys, translations, or screenshots. You can use them as
+an authentication method for Tolgee integration libraries.
+
+The Project API key
+ - Authorizes access to single project resources
+ - Allows specifying permitted scopes
+
+You can manage all your API keys via the `Project API keys` section in the user account menu.
+To access your API keys, open user menu (Avatar in Top right) and select `Project API keys`.
+
+In this section you are able to generate, edit, delete or regenerate your keys.
+
+### Generation
+
+Project API keys can be generated by accessing user menu (Avatar in Top right) > `Project API keys` > `+ ADD` button.
+
+Then you can generate a project API by filling in required fields and selecting scopes.
+
+![API key generation](/img/docs/platform/project_api_key_generation.png)
+
+### Regeneration
+If you want to obtain a new Project API key value, you can regenerate it by clicking the `REGENERATE` button. Then you can select new
+expiration date.
+
+### Usage
+
+To access REST API endpoints with a Project API key, you can provide the API key via the `X-API-Key` header or via the `ak` query param.
+
+Example using header:
+```
+curl "https://app.tolgee.io/v2/api-keys/current" -H "X-API-Key: tgpak_mmztgz2hmaxts2rynu3w6zlkobsksksnfvg6mlrgq2gc2dq"
+```
+
+Example using query param:
+```
+curl "https://app.tolgee.io/v2/api-keys/current?ak=tgpak_mmztgz2hmaxts2rynu3w6zlkobsksksnfvg6mlrgq2gc2dq"
+```
```

Comment:
Considering `ak` is scheduled for deprecation it might be best to not document the `ak` query parameter and encourage the use of the X-API-Key header.

Reply:
I am not sure about deprecating it yet. But we can say it't not recommended.